packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -414,7 +420,9 @@ async function restartSetup() {
 {#if displayCancelSetup}
   <!-- Create overlay-->
   <div class="fixed top-0 left-0 right-0 bottom-0 bg-black bg-opacity-60 bg-blend-multiply h-full grid z-50">
-    <div class="flex flex-col place-self-center w-[550px] rounded-xl bg-charcoal-800 shadow-xl shadow-black">
+    <div
+      class="flex flex-col place-self-center w-[550px] rounded-xl bg-charcoal-800 shadow-xl shadow-black"
+      aria-label="Skip setup popup">
```

Comment:
here it's now using UpperCase on every words like the others `Onboarding Status Message` or `Step Buttons`


but still I'm wondering why we need the aria-label there ?

should it be on the button that don't have an aria-label for now ?
```
<Button type="secondary" aria-label="Cancel" on:click="{() => setDisplayCancelSetup(false)}">Cancel</Button>
<Button type="primary" class="mr-2" on:click="{() => cancelSetup()}">Ok</Button>
```

Reply:
Thanks for the comment, I fixed the Upper Case for the rest of the labels in the file as well. 

I think it's good to label the popup message for testing purposes so it's clear what we're working with. 

It's also not needed to label buttons when their name is visible in the ui, because equally the aria-label and the button name can be used for the name in getByRole("button", {name:xxx}).